packages/components/organisms/f-checkout/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-checkout",
   "description": "Fozzie Checkout – Fozzie Checkout Component",
-  "version": "0.60.0",
+  "version": "0.62.0",
```

Comment:
Did you mean to skip a minor version?

Reply:
Yes, because there's another PR about to be merged which is version 61 😄

Reply:
I see!